src/Features/CSharp/Portable/ConvertNamespace/ConvertNamespaceHelper.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.ConvertNamespace
+{
+    internal static class ConvertNamespaceHelper
+    {
+        internal static bool CanOfferUseRegular(OptionSet optionSet, BaseNamespaceDeclarationSyntax declaration, bool forAnalyzer)
+        {
+            if (declaration is not FileScopedNamespaceDeclarationSyntax)
+                return false;
+
+            var option = optionSet.GetOption(CSharpCodeStyleOptions.PreferFileScopedNamespace);
+            var userPrefersRegularNamespaces = !option.Value;
+            var analyzerDisabled = option.Notification.Severity == ReportDiagnostic.Suppress;
```

Comment:
Same comment here. `Notification.Severity` most likely doesn't reflect the `dotnet_diagnostic.....` syntax

Reply:
i reaklly do not want our features to deviate from each other.  until there is a suitable new style we should keep our code consistent.  Otherwise we'll just have a hodgepodge of impls with varying patterns that people have to udnerstand.  If and when we have some new pattern that everything should follow, we can move all our code to that at once.